charts/blackduck-connector/values.yaml

```diff
@@ -0,0 +1,153 @@
+# Default values for Opssight.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+# Docker registry to pull Black Duck images
+registry: docker.io/blackducksoftware
+imageTag: 2.2.5
+
+# image pull secret to download the Black Duck images (mostly applicable for air gapped customers)
+imagePullSecrets: [] # array of strings delimited by comma
+
+# This is used to start or stop the app. Set to Running to start, or Stopped to stop.
+status: Running
+
+logLevel: "debug"
+
+# External Hub ~>
+#  {
```

Comment:
These examples are json and also an external Black Duck is an array

Reply:
I think operator kinda did that conversion from array to map internally, cos I tried array and it won't work. It needs a map

Reply:
pls provide an example in yaml format